libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/DBSchemaAccessor.java

```diff
@@ -218,6 +218,16 @@ default List<String> showExternalTables(String schemaName) {
      */
     List<DBTableColumn> listBasicViewColumns(String schemaName, String viewName);
 
+    /**
+     * Get all external table columns(hold only basic info) in the specified schema
+     */
+    Map<String, List<DBTableColumn>> listBasicExternalTableColumns(String schemaName);
```

Comment:
I think `listExternalTableColumns` is enough.

Reply:
![image](https://github.com/user-attachments/assets/1a02c0bf-4beb-4a20-98f3-8aef2658b120)
I recommend keeping the current method name the same as table and view. Only a small portion of the column object's field information is retrieved.
In fact, the implementation of the above three methods is to share the same sql statement, but now it is drawn into three methods and repeated writing three of the same sql, is it necessary to make an interface method?